NEWS.md

```diff
@@ -1,8 +1,4 @@
 # admiralmetabolic 0.1.0
 
-- Initial package release focused on `<<ADD INITIAL SCOPE>>`
-
-Note that after first release you should then start maintaining this `NEWS.md` file for every user-facing update.
-You should categorize the updates under title sections for New Features, Updates of Existing Functions,
-Breaking Changes, Documentation, Various.
-Each update should also reference the original issue by adding at the end: (#ISSUE-NUMBER)
+- Initial package release mainly focused on obesity indicators
+(e.g. Waist to Hip Ratio, Waist to Height Ratio).
```

Comment:
```suggestion
(e.g. functions for Waist to Hip Ratio, Waist to Height Ratio).
```

Reply:
I think if we are going to mention the functions then we should mention our templates and vignettes too. Alternatively as this is our first release we could omit mentioning anything like that in the changelog at all. @AndersAskeland what do you think?

Reply:
ok, happy to go with this

Reply:
@kiwi5104679 What are your thoughts on this?

Reply:
@AndersAskeland , @manciniedoardo , thank you.
I think it's better to have subsections.
I'll add "Functions" and "Documentation".

Reply:
Hi @AndersAskeland , @manciniedoardo.
I've updated NEWS.md.
I noticed that {admiral} and other related packages use "New Features" subsection to describe new functions instead of "Functions", so I added "New Features" subsection in {admiralmetabolic}.
Please let me know if it's better to use "Functions".
![image](https://github.com/user-attachments/assets/42182f79-bfac-4b3e-bac9-f08c7fd5b8aa)
![image](https://github.com/user-attachments/assets/8b71aa47-5473-4344-b172-0c768a4a808c)

Reply:
In addition, should we have hyperlinks in changelog?
![image](https://github.com/user-attachments/assets/110536b5-85db-42bc-914d-38979224952e)

It's user friendly to have hyperlinks, but I'm concerned that maintenance may be difficult.

Reply:
Exactly.
Updated subsection to "Functions".